### ISSUE TITLE
[Requests] Bookmark auto-download & Bato.to things

### ISSUE THREAD

**Issue by Demorra** [NguyenDanPhuong/MangaRipper#71]
Just some ideas that would be useful to what I'd assume to be many others like me...
I read many ongoing manga. Individually checking the bookmarks, selecting the download path, and selecting the new available chapters for each series is rather time consuming. I understand that this is a free software and what I'm asking for might be a lot, but just if it's possible:

- Automatically downloading chapters into a series' folder - I know you said this is low priority since it  can be done by the end-user, but it's a quality of life thing that affects people who download multiple series. 

- Automatically checking bookmarks and download new chapters

- Adding a language filter to Bato.to - This isn't really an issue if you download chapters manually, but I think it might be necessary if you were to implement an auto-download feature, and it's a minor QoL improvement even if you were to download them manually.

- Individual bato.to login - A place within the app to fill in the user's login credentials instead of using that "communal" account. I think of this as more of a safety/privacy thing. I know we can change it from a file ourselves, so it might not be a priority for you. I guess for now maybe just add a how to on the wiki

**Comment by NguyenDanPhuong** [comment]
Thank for your suggestion. It's a nice feature list. I will keep track and will do it.

**Comment by GambitKZ** [comment]
>Individual bato.to login - A place within the app to fill in the user's login credentials instead of using that "communal" account. 

Wait. The thing is, you **should** use your own login/pass for Batoto. That's why we have "MangaRipper.Configuration.json" file.

**Comment by Demorra** [comment]
> Wait. The thing is, you should use your own login/pass for Batoto. That's why we have "MangaRipper.Configuration.json" file.

Yes, I understand this, but if MangaRipper were to become more popular and widespread, how many people do you think will? If you leave in a communal account in the first place, that account may become compromised, so it'd be best to just have a gui within the app's interface to put in your own credentials and leaving the .json files empty.

But eh, maybe I'm over thinking it.

**Comment by GambitKZ** [comment]
Well, this account is used mostly for passing unit Tests. However, I agree that no one will change the password if it's already working out of the box.
Not sure how to fix this issue.
